File: JS-Advanced-October-2016/16. Modules-Babel-CommonJS-Exercises/Exercises/People-And-Posts/js/person.js
"use strict";

class Person {
    constructor(name, age) {
        this.age = age;
        this.name = name;
    }

    addToSelector(selector) {
        let result = $('<div>')
            .addClass('person ' + this.name)
            .append($('<p>')
                .addClass('name')
                .text(this.name))
            .append($('<p>')
                .addClass('age')
                .text(this.age))
            .append($('<div>')
                .addClass('posts ' + this.name));
        $(selector).append(result);
    }
}

module.exports = Person;

File: C#-OOP-Basics-June-2017/02. Defining-Classes-Exercises/Exercises/Creating-Constructors/Person.cs
public class Person
{
    public Person(string name, int age)
    {
        this.Name = name;
        this.Age = age;
    }

    public Person(int age)
        : this("No name", age)
    {
    }

    public Person()
        : this("No name", 1)
    {
    }

    public string Name { get; }

    public int Age { get; }
}